generator/internal/sample_generator.cc

```diff
@@ -177,6 +177,69 @@ void SetRetryPolicy(std::vector<std::string> const& argv) {
 }
 )""");
 
+  if (HasLongrunningMethod()) {
+    HeaderPrint(R"""(
+void SetPollingPolicy(std::vector<std::string> const& argv) {
+  if (!argv.empty()) {
+    throw google::cloud::testing_util::Usage{"set-client-policy-policy"};
+  }
+  //! [set-polling-policy]
+
+  // The polling policy controls how the client waits for long-running
+  // operations. `GenericPollingPolicy<>` combines existing policies.
+  // In this case, keep polling until the operation completes (with success
+  // or error) or 45 minutes, whichever happens first. Initially pause for
+  // 10 seconds between polling requests, increasing the pause by a factor
+  // of 4 until it becomes 2 minutes.
+  auto options = google::cloud::Options{}
+  .set<google::cloud::$product_namespace$::$service_name$PollingPolicyOption>(
+    google::cloud::GenericPollingPolicy<google::cloud::$product_namespace$::$service_name$LimitedTimeRetryPolicy,google::cloud::ExponentialBackoffPolicy>(
+     google::cloud::$product_namespace$::$service_name$LimitedTimeRetryPolicy(
+          /*maximum_duration=*/std::chrono::minutes(45)),
+      google::cloud::ExponentialBackoffPolicy(
+          /*initial_delay=*/std::chrono::seconds(10),
+          /*maximum_delay=*/std::chrono::minutes(2),
+          /*scaling=*/4.0))
+      .clone());
```

Comment:
Consider:

```suggestion
  auto options = google::cloud::Options{}
  .set<google::cloud::$product_namespace$::$service_name$PollingPolicyOption>(
    google::cloud::GenericPollingPolicy<
        google::cloud::$product_namespace$::$service_name$RetryPolicy,
        google::cloud::BackoffPolicy>(
     google::cloud::$product_namespace$::$service_name$LimitedTimeRetryPolicy(
          /*maximum_duration=*/std::chrono::minutes(45)),
      google::cloud::ExponentialBackoffPolicy(
          /*initial_delay=*/std::chrono::seconds(10),
          /*maximum_delay=*/std::chrono::minutes(2),
          /*scaling=*/4.0))
      .clone());
```

Or maybe introduce some type aliases?